### PR TITLE
Awesome Books: Full Website

### PR DESCRIPTION
**Pull request by AduwuJoseph**
In this project, I and my partner take the Awesome books application and converted it into a fully functional website. You will refactor the project to add three new sections on top of the two sections you have already completed: Books list (done), Add book form (done), a new Contact info section, a Navigation bar, and a Footer.
We achieved this by:

We Modified the Awesome books application to have:
    A Navigation bar.
    Three content sections:
        Books list.
        Add book form.
        Contact info.

We make sure that when you click on any of the links, the main area changes to display only one of the three content sections.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -82,3 +108,11 @@ deleteBookObj.forEach((trigger) => {
     ab.deleteBooks(trigger.dataset.bookid);
   });
 });
+
+const currentDate = () => {
+  const newDate = new Date();
+  date.innerHTML = `${newDate.toDateString()}, ${newDate.getHours()}:${newDate.getMinutes()}:${newDate.getSeconds()}pm `;
+  setTimeout(currentDate, 1000);
+};
```
Comment (codepantha) [inline script.js:116]: - [x] Well done for implementing the current date and time feature 👏 it's impressive 🍷 . The only thing that's left is to align it to the right just as in the wireframe design.

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️ 

## Hi team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Used GitFlow
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AduwuJoseph** [comment]
I have fixed the required changes @codepantha 
Waiting for your approval pls

**Review by mwanawabangona — APPROVED** [review]
Hi @AduwuJoseph  ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implimenting the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**